hw/bsp/ci40/src/arch/mips/abiflags.S

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Do you have license blurb for this file?

Reply:
Fixed to be the same as the others, thanks.